### ISSUE TITLE
Cannot Train POS with Locale Other Than English

### ISSUE THREAD

**Issue by schrieveslaach** [brendano/ark-tweet-nlp#45]
If I train the arktweet POS tagger with e.g. German locale (cf. [train method](https://github.com/brendano/ark-tweet-nlp/blob/master/src/cmu/arktweetnlp/Train.java#L185)), the training process fails because it generates a file containing decimals with German formatting. For example, numbers like `0.2` are formatted as `0,2` (German notation) and the trainer component fails to load this file because of the comma.

**Comment by brendano** [comment]
Hm. Maybe the locale has to be set as a java flag or property?  Or it would be better to use non-locale-dependent, floating point number parsing code.  I have no idea how to do that in java.

**Comment by schrieveslaach** [comment]
I agree to use a locale independent format. For example, you could write number in English format. 

```java
public class I18NTester {
   public static void main(String[] args) {
      String pattern = "###.##";
      double number = 123.45;

      Locale enlocale  = new Locale("en", "US");
   
      DecimalFormat decimalFormat = (DecimalFormat) NumberFormat.getNumberInstance(enlocale);
      decimalFormat.applyPattern(pattern);

      System.out.println(decimalFormat.format(number));
   
      }
}
```
